packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -505,11 +504,32 @@ export const transformPlotlyJsonToGaugeProps = (
   };
 };
 
+const MAX_DEPTH = 8;
```

Comment:
how did we determine the max_depth of 8?

Reply:
Our data generally has a depth of 5/6. Adding 2 more levels to be on safe side.